src/kafka_helpers.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Collection of helper functions to be called by KafkaCharm event handlers."""
+
+import logging
+from typing import Dict
+
+from charms.operator_libs_linux.v0 import apt
+from charms.operator_libs_linux.v1 import snap
+
+logger = logging.getLogger(__name__)
+
+
+def install_packages():
+    """Loads the Kafka snap from LP."""
```

Comment:
missing `Raises` section in the docstring - important especially since the exceptions are handled where this function is called